internal/exec/exec.go

```diff
@@ -5,8 +5,10 @@ import (
 	"fmt"
 	"io"
 	"os/exec"
+	"strings"
 	"time"
 
+	"github.com/kballard/go-shellquote"
 	"github.com/pkg/errors"
 
 	"github.com/SAP/cloud-mta-build-tool/internal/logs"
```

Comment:
imports sequence

Reply:
Isn't it in the correct place? It's an open source